### ISSUE TITLE
Dropout: A Simple Way to Prevent Neural Networks from Overfitting

### ISSUE THREAD

**Issue by youjun99** [GDSC-Ewha-4th/Study-paperReview#59]
Srivastava, Nitish, et al. "Dropout: a simple way to prevent neural networks from overfitting." Journal of Machine Learning Research 15.1 (2014): 1929-1958.

<Dropout>
-prevents units from co-adapting too much
-samples from an exponential number of different “thinned” networks

Deep neutral networks;
-multiple non-linear hidden layers
-> can learn complicated relationships
-> BUT ALSO *sampling noise *overfitting
-> solution?
	: regularization, soft weight sharing
	*approximating an equally weighted geometric mean of the predictions of an exponential number of learned models that share parameters

DROPOUT addresses both issues!
-lower generalization of error, on a wide variety of classification problems

*X limited to feed-forward neural nets,
-can be applied to graphical models 
Such as Boltzmann Machines
*dropout Restricted Boltzmann Machine model vs. standard RBM
-take large model that overfits easily & repeatedly sample and train sub-models from it

*drawbacks of dropout
-increased training time (2~3 times longer)
=> how to obtain benefits of dropout w/o stochasticity?
: marginalize noise to obtain a regularizer!
BUT still not obvious how to obtain an equivalent regularizer for more complicated models, else than L2 regularization
